src/igvfd/schemas/treatment.json

```diff
@@ -67,6 +70,54 @@
             ],
             "comment": "temperature must be specified if temperature_units is specified"
         },
+        "amount": {
```

Comment:
I think the logic should be reversed. "if depletion is True - then amount and units should not be specified"